corporate/lib/stripe.py

```diff
@@ -1967,11 +1973,13 @@ def approve_sponsorship(self) -> None:
         # TBD
         pass
 
+    @override
+    def is_sponsored(self) -> bool:
+        return self.remote_realm.plan_type == self.remote_realm.PLAN_TYPE_COMMUNITY
```

Comment:
We have a local variable in `do_change_plan_type` with the same name that this will interfere with; needs some adjustment.

Reply:
I realized this isn't actually a conflict, as one is `self.is_sponsored()` and the other just `is_sponsored`.